cluster/common.go

```diff
@@ -28,14 +30,22 @@ type CommonCluster interface {
 	DeleteCluster() error
 	UpdateCluster(*bTypes.UpdateClusterRequest) error
 	GetID() uint
+	GetSecretID() string
 	GetModel() *model.ClusterModel
 	CheckEqualityToUpdate(*bTypes.UpdateClusterRequest) error
 	AddDefaultsToUpdate(*bTypes.UpdateClusterRequest)
 	GetAPIEndpoint() (string, error)
 	DeleteFromDatabase() error
-	//ModifyCluster(*model.ClusterModel)
-	//GetKubernetesConf()
-	//GetKubernetesEndpoint()
+	GetOrg() uint
+}
+
+func GetSecret(cluster CommonCluster) (*secret.SecretsItemResponse, error) {
+	org := strconv.FormatUint(uint64(cluster.GetOrg()), 10)
+	secret, err := secret.Store.Get(org, cluster.GetSecretID())
+	if err != nil {
```

Comment:
this can be simplified, by just returning the secret.Store.Get(org, cluster.GetSecretID())